xformers/ops/fmha/triton_splitk.py

```diff
@@ -1028,19 +1028,31 @@ def not_supported_reasons(cls, d: Inputs) -> List[str]:
     @classmethod
     def get_split_k(cls, B: int, G: int, H: int, Mk: int) -> int:
         """Heuristic for the number of splits"""
+        print(f"B = {B}, G = {G}, H = {H}, Mk = {Mk}")
```

Comment:
nit: remove.

Reply:
Sorry, it seems like I pushed an unexpected commit to this PR. I just convert this PR to draft. will let you know when I cleaned up the code.